src/pkg/podman/podman.go

```diff
@@ -280,6 +282,47 @@ func Inspect(typearg string, target string) (map[string]interface{}, error) {
 	return info[0], nil
 }
 
+func InspectSkopeo(typearg string, target string) (interface{}, error) {
+
+	var stdout bytes.Buffer
+
+	args := []string{"inspect", target}
+
+	if err := shell.Run("skopeo", nil, &stdout, nil, args...); err != nil {
+		return nil, err
+	}
+
+	output := stdout.Bytes()
+
+	var info interface{}
+
+	if err := json.Unmarshal(output, &info); err != nil {
```

Comment:
... then we can have a custom type called `Image` that implements the [json.Unmarshaler](https://pkg.go.dev/encoding/json#Unmarshaler) interface.  [This](https://go.dev/blog/json) gives a good introduction to handling JSON in Go.

There's a similar `Image` type in `src/pkg/podman/podman.go` that's used to parse JSON in `GetImages()`.  We could have used this same type to parse Skopeo's JSON, but it's different from Podman's JSON, so it might be better to keep them separate.